### PR TITLE
Use Rollbar to manage application errors for Rails

### PR DESCRIPTION
**Pull request by tahb**
The Rails Template PR is already open and can be found here https://github.com/dxw/rails-template/pull/35

### CODE REVIEW

File: rfc-020-use-rollbar-to-monitor-rails-errors.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ projects.
+
+## Problem
+
+We need to use a tool that collects errors that happen across our live
+ environments.
+We need a tool that can notify the team of new or important errors.
+We need a way to record and share the outputs of our investigations and any
+ plans to tackle the problem, more investigation, intended fixes etc.
+We need a way to group common issues.
```
Comment (erbridge) [inline rfc-020-use-rollbar-to-monitor-rails-errors.md]: Did you want these to be separate paragraphs? If so, they need a blank line in between them.

File: rfc-020-use-rollbar-to-monitor-rails-errors.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Proposal
+
+We SHOULD use Rollbar as our default tool for all Rails projects.
+We MAY use another service if the client has a preference, this MUST be captured
+ in an ADR within the project itself.
```
Comment (erbridge) [inline rfc-020-use-rollbar-to-monitor-rails-errors.md]: Same paragraph comment as above.

File: rfc-020-use-rollbar-to-monitor-rails-errors.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Next steps
+
+Add Rollbar to the rails-template repository
+ <https://github.com/dxw/rails-template/>
```
Comment (erbridge) [inline rfc-020-use-rollbar-to-monitor-rails-errors.md]: Can we make this a link instead of putting the URL?

Comment (tahb) [reply rfc-020-use-rollbar-to-monitor-rails-errors.md]: I just did `npm lint:fix` and trusted that it would sort it all 🤷‍♂

Comment (erbridge) [reply rfc-020-use-rollbar-to-monitor-rails-errors.md]: Linter doesn't know what your intentions are :).

**Comment by erbridge** [comment]
Fixed up the formatting. Hope that's ok.

**Pull request merged**